capa/render/result_document.py

```diff
@@ -24,7 +24,6 @@
 
 class FrozenModel(BaseModel):
     class Config:
-        frozen = True
```

Comment:
lets not call this `Frozen` if its not frozen.

for things that can still stay frozen, lets use the existing `FrozenModel`. but lets also introduce a `Model` class for things that aren't frozen.